### DISCUSSION TITLE
Merging changes from this template

### DISCUSSION THREAD

**Discussion by dreaquil** [ashleve/lightning-hydra-template#491] [General]
Hi @ashleve,

First off, thanks for your work on such a fantastic template. I've been using it for around a year now. I have a question about your recommendations for updating projects based on this template. My process is usually to merge in the latest release and resolve conflicts but I don't do this every release. I'm currently updating and would like to get all the latest changes to save me time down the line. Would you say that the main branch is in a functional state or would you recommend only merging in the release? If so, do you have an expectation of when v1.5.0 will be released?

**Comment by ashleve** [comment]
@dreaquil Thank you for using the project!

The `main` branch is supposed to be always stable and updated to recent torch/lightning/hydra releases so I would recommend pulling the changes from there. 

To be honest with you, most of the time I don't really plan releases - I make them once in a couple of months to point out what has been changing so I don't think there's much point in waiting for the release. I will probably mark the current main branch as v1.5.0 this month.
